src/GRA.Web/Areas/MissionControl/Views/Home/Index.cshtml

```diff
@@ -121,11 +121,22 @@ else
                 </div>
                 <div class="panel-body">
                     <ul class="nav nav-pills nav-stacked">
-                        @foreach (var category in Model.NewsCategories)
```

Comment:
If you extend `Domain.Model.NewsCategory` to have a bool of `IsNew` you can use the `foreach` here and skip all the `.ElementAt()` stuff.